src/core_plugins/metrics/public/kbn_vis_types/request_handler.js

```diff
@@ -9,11 +11,20 @@ const MetricsRequestHandlerProvider = function (Private, Notifier, config, timef
     name: 'metrics',
     handler: function (vis /*, appState, uiState, queryFilter*/) {
 
+      let timezone = config.get('dateFormat:tz', 'Browser');
```

Comment:
`src/ui/public/vis/lib/timezone`?

fwiw, the folder structure of visualize needs work. we're thinking of consolidating. https://github.com/elastic/kibana/issues/12675.